x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwtValidateUtil.java

```diff
@@ -67,30 +67,24 @@ public static void validate(
         final List<JWK> jwks
     ) throws Exception {
         final Date now = new Date();
-        LOGGER.debug(
-            "Validating JWT, now ["
-                + now
-                + "], alg ["
-                + jwt.getHeader().getAlgorithm()
-                + "], issuer ["
-                + jwt.getJWTClaimsSet().getIssuer()
-                + "], audiences ["
-                + jwt.getJWTClaimsSet().getAudience()
-                + "], typ ["
-                + jwt.getHeader().getType()
-                + "], auth_time ["
-                + jwt.getJWTClaimsSet().getDateClaim("auth_time")
-                + "], iat ["
-                + jwt.getJWTClaimsSet().getIssueTime()
-                + "], nbf ["
-                + jwt.getJWTClaimsSet().getIssueTime()
-                + "], exp ["
-                + jwt.getJWTClaimsSet().getExpirationTime()
-                + "], kid ["
-                + jwt.getHeader().getKeyID()
-                + "], jti ["
-                + jwt.getJWTClaimsSet().getJWTID()
-        );
+
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug(
+                "Validating JWT, now [{}], alg [{}], issuer [{}], audiences [{}], typ [{}],"
+                    + " auth_time [{}], iat [{}], nbf [{}], exp [{}], kid [{}], jti [{}]",
+                now,
+                jwt.getHeader().getAlgorithm(),
+                jwt.getJWTClaimsSet().getIssuer(),
+                jwt.getJWTClaimsSet().getAudience(),
+                jwt.getHeader().getType(),
+                jwt.getJWTClaimsSet().getDateClaim("auth_time"),
+                jwt.getJWTClaimsSet().getIssueTime(),
+                jwt.getJWTClaimsSet().getIssueTime(),
```

Comment:
```suggestion
                jwt.getJWTClaimsSet().getNotBeforeTime(),
```

I just noticed this bug. You didn't introduce it, but you have an opportunity to fix it in this PR. Can you make this change?